scripts/flytekit_install_spark3.sh

```diff
@@ -42,13 +43,14 @@ rm -rf spark-dist.tgz
 rm -rf spark-dist
 
 # Fetch Hadoop Distribution with AWS Support
-wget http://apache.mirrors.tds.net/hadoop/common/hadoop-2.7.7/hadoop-2.7.7.tar.gz -O hadoop-dist.tgz
-echo 'd129d08a2c9dafec32855a376cbd2ab90c6a42790898cabbac6be4d29f9c2026  hadoop-dist.tgz' | sha256sum --check
+#wget http://apache.mirrors.tds.net/hadoop/common/hadoop-2.7.7/hadoop-2.7.7.tar.gz -O hadoop-dist.tgz
```

Comment:
remove comments?

Reply:
ohh this file should not be part of the update, removing